x-pack/plugins/ml/public/application/components/job_messages/job_messages.tsx

```diff
@@ -22,6 +23,12 @@ import { blurButtonOnClick } from '../../util/component_utils';
 import { JobIcon } from '../job_message_icon';
 import { useEnabledFeatures } from '../../contexts/ml';
 
+const cssOverride = css({
+  '.euiTable': {
+    backgroundColor: 'transparent',
```

Comment:
By overriding this in the actual component, it means it's transparent for both places where it's used.
IMO in the AD jobs list it shouldn't be transparent.

![image](https://github.com/user-attachments/assets/4200ebf5-e389-43a1-a5b4-4228ae9cbc1b)

vs

![image](https://github.com/user-attachments/assets/a3448aea-34d9-4b05-8962-9250c4496f70)


Also it appears the padding has changed. Not a huge problem, I'm not sure which looks better.

Reply:
Yeah, apparently there's no direct way to set the background for EuiInMemoryTable so working out the best way to apply it the way we want. I actually prefer less padding but that's just me 😆